app/move/views/view.njk

```diff
@@ -60,12 +60,53 @@
   </header>
 {% endblock %}
 
+{% set urls = {
+  personal_details: 'personal-details',
+  move_details: 'move-details',
+  court: 'court-information',
+  risk: 'risk-information',
+  health: 'health-information'
+} %}
+
+{% macro updateLink(category, categoryHeading) %}
+  {% if canAccess("move:update") %}
+    {% if not categoryHeading %}
+      {% set categoryKey = "moves::assessment.categories." + category + ".heading" %}
+      {% set stepsKey = "moves::steps." + category + ".heading" %}
+      {% set categoryHeading = t([categoryKey, stepsKey]) %}
+    {% endif %}
+    {% set updateLinkText = t("moves::assessment.categories.update_link", {
+      context: "with_description" if categoryHeading,
+      category: categoryHeading
+    }) %}
+    <p class="update-link"><a href="/move/{{ move.id }}/edit/{{ urls[category] }}" data-update-link="{{ category }}">{{ updateLinkText | safe }}</a></p>
+  {% endif %}
+{% endmacro %}
+
 {% block contentMain %}
+<style>
+.app-section {
+  position: relative;
+}
+.update-link {
+  position: absolute;
+  top: 0.25rem;
+  right: 0;
+}
+.sticky-sidebar .update-link {
```

Comment:
It feels like rather than overriding the style for that link when its within a certain element, could we either not apply the class in the first place that sets the element to `position: absolute` or apply a BEM style modifier to position the element statically?

Reply:
Taken care of by addition of explicit positioning classes